tests/acquia_lift.test

```diff
@@ -0,0 +1,561 @@
+      'acquia_lift_field_mappings[content_section]' => 'taxonomy_context__tags',
+      'acquia_lift_field_mappings[content_keywords]' => 'taxonomy_context__tags',
+      'acquia_lift_field_mappings[persona]' => 'taxonomy_context__tags',
+    );
+    $this->drupalPost('admin/config/services/acquia_lift', $edit, $submit_button_text);
+    $this->drupalGet('node/' . $this->articles['one_tag']->nid);
+    $expected_tag = $this->tags['one_tag'][0];
+    $this->assertLiftMetaTag('content_section', $expected_tag);
+    $this->assertLiftMetaTag('content_keywords', $expected_tag);
+    $this->assertLiftMetaTag('persona', $expected_tag);
+
+    $this->drupalGet('node/' . $this->articles['two_tags']->nid);
+    $expected_tags = implode(',', $this->tags['two_tags']);
+    $this->assertLiftMetaTag('content_section', $expected_tags);
+    $this->assertLiftMetaTag('content_keywords', $expected_tags);
+    $this->assertLiftMetaTag('persona', $expected_tags);
+
+    $edit = array(
+      'acquia_lift_field_mappings[content_section]' => 'taxonomy_context__tags',
+      'acquia_lift_field_mappings[content_keywords]' => '',
+      'acquia_lift_field_mappings[persona]' => 'taxonomy_context__tags',
+    );
+    $this->drupalPost('admin/config/services/acquia_lift', $edit, $submit_button_text);
+    $this->drupalGet('node/' . $this->articles['one_tag']->nid);
+    $expected_tag = $this->tags['one_tag'][0];
+    $this->assertLiftMetaTag('content_section', $expected_tag);
+    $this->assertNoLiftMetaTag('content_keywords');
+    $this->assertLiftMetaTag('persona', $expected_tag);
+  }
+
+  function testUDFMappings() {
+    $this->configureLift();
+    $this->drupalLogin($this->admin_user);
+    $edit = array(
+      'acquia_lift_udf_mappings[person][person_udf1]' => 'taxonomy_context__tags',
+      'acquia_lift_udf_mappings[touch][touch_udf20]' => 'taxonomy_context__tags'
+    );
+    $this->drupalPost('admin/config/services/acquia_lift', $edit, $this->admin_button_text);
+    $this->drupalGet('node/' . $this->articles['one_tag']->nid);
+    $expected_tag = $this->tags['one_tag'][0];
+    $this->assertLiftMetaTag('person_udf1', $expected_tag);
+    $this->assertLiftMetaTag('touch_udf20', $expected_tag);
+
+    $edit = array(
+      'acquia_lift_udf_mappings[person][person_udf1]' => 'taxonomy_context__tags',
+      'acquia_lift_udf_mappings[touch][touch_udf20]' => '',
+      'acquia_lift_udf_mappings[event][event_udf5]' => 'taxonomy_context__tags',
+    );
+    $this->drupalPost('admin/config/services/acquia_lift', $edit, $this->admin_button_text);
+    $this->drupalGet('node/' . $this->articles['two_tags']->nid);
+    $expected_tags = implode(',', $this->tags['two_tags']);
+    $this->assertLiftMetaTag('person_udf1', $expected_tags);
+    $this->assertNoLiftMetaTag('touch_udf20');
+    $this->assertLiftMetaTag('event_udf5', $expected_tags);
+  }
+
+  /**
+   * Helper function to configure acquia_lift settings.
+   */
+  protected function configureLift() {
```

Comment:
Not interested in getting into a discussion on this one for sure.  Things can always be changed when they need to.  https://github.com/acquia/acquia_lift/pull/245/commits/1bcf876abff2adc26a92f293e8c68927bf5b3696